src/core/service.c

```diff
@@ -2937,7 +2937,7 @@ static int service_deserialize_item(Unit *u, const char *key, const char *value,
 
                 r = extract_first_word(&value, &fdn, NULL, EXTRACT_CUNESCAPE | EXTRACT_UNQUOTE);
                 if (r <= 0) {
-                        log_unit_debug_errno(u, r, "Failed to parse fd-store-fd value \"%s\": %m", value);
+                        log_unit_debug(u, "Failed to parse fd-store-fd value: %s", value);
```

Comment:
needs zerook, no?

Reply:
Hmm, `Failed to parse fd-store-fd value "123": Success` sounds strange. Another option is call `log_unit_debug_errno()` for `r < 0` and `log_unit_debug()` for `r == 0`. But I think that is mostly redundant...

Reply:
true